### DISCUSSION TITLE
How to select the detected objects class with torch.hub?

### DISCUSSION THREAD

**Discussion by tgrmev** [ultralytics/yolov5#10654] [Q&A]
I am calling yolo yolov5m.pt model with this code. I only want to detect the people from my environment. For this aim what should I add to this line ?
`model = torch.hub.load("./yolov5", 'custom', path="./yolov5/yolov5m.pt", source='local')`

I tried with this :  `model = torch.hub.load("./yolov5", 'custom', path="./yolov5/yolov5m.pt", source='local' , classes = 0 )` and then I got **_TypeError: custom() got an unexpected keyword argument 'classes'_** error. Is it not possible to select the detected class, if we work with custom model?

**Comment by Robotatron** [comment]
Do a prediction first, then select your class.

**Reply by tgrmev** [reply]
Could you write the line of that code after `model = torch.hub.load("./yolov5", 'custom', path="./yolov5/yolov5m.pt", source='local')` ?
